packages/router-vite-plugin/src/index.ts

```diff
@@ -3,29 +3,16 @@ import { join, normalize } from 'path'
 import { readFile } from 'fs/promises'
 import {
   type Config,
-  configSchema,
+  getConfig,
   generator,
 } from '@tanstack/router-generator'
 
 const CONFIG_FILE_NAME = 'tsr.config.json'
 
 type UserConfig = Partial<Config>
 
-async function readConfigFile(path: string): Promise<UserConfig> {
-  try {
-    const raw = await readFile(path, 'utf-8')
-    return JSON.parse(raw) as UserConfig
-  } catch {
-    return {} as UserConfig
-  }
-}
-
-async function buildConfig(config: UserConfig, root: string): Promise<Config> {
-  const fileConfig = await readConfigFile(join(root, CONFIG_FILE_NAME))
-  return configSchema.parse({
-    ...fileConfig,
-    ...config,
-  })
+async function buildConfig(inlineConfig: UserConfig): Promise<Config> {
```

Comment:
can't we just call `getConfig` down there and remove this function?

Reply:
you are right, of course